src/main/java/nextstep/ladder/Model/People.java

```diff
@@ -0,0 +1,64 @@
+package nextstep.ladder.Model;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class People {
+
+    private static final String MIN_MEMBER_ERROR = "게임을 하려면 최소 2명이어야 합니다.";
+    private static final String NOT_CONTAIN_PERSON = "일치하는 사용자가 존재하지 않습니다.";
+    private static final int STANDARD_GAME = 2;
+    private final List<Person> people;
+
+    public People(List<Person> people) {
+        validate(people);
+        this.people = people;
+    }
+
+    public void offerReward(Ladder ladder, Rewards rewards) {
+        List<Reward> Rewards = IntStream.range(0, people.size())
+                .map(it -> ladder.move(it))
+                .mapToObj(it -> rewards.getRewardByLocation(it))
+                .collect(Collectors.toList());
```

Comment:
```suggestion
        List<Reward> Rewards = IntStream.range(0, people.size())
                .map(ladder::move)
                .mapToObj(rewards::getRewardByLocation)
                .collect(Collectors.toList());
```
조금 더 간단히 작성해 볼 수 있을 것 같습니다!

Reply:
너무 감사드립니다.!! 이 부분에 대해서 수정한 이후에 다음에도 이 방법을 적극 활용하겠습니다.!